frontend/src/views/eligibility/CoverageStatusCheck.vue

```diff
@@ -22,9 +22,8 @@
           <p>Select by clicking one or more boxes</p>
         </AppCol>
         <AppCol>
-          <AppCheckbox :errorValue="v$.checkSubsidyInsuredService" id="checkSubsidyInsuredService" label="Check for Subsidy Insured Service" v-model="checkSubsidyInsuredService"/>
-          <AppCheckbox :errorValue="v$.checkLastEyeExam" id="checkLastEyeExam" label="Check for Last Eye Exam" v-model="checkLastEyeExam"/>
-          <AppCheckbox :errorValue="v$.checkPatientRestriction" id="checkPatientRestriction" label="Check for Patient Restriction" v-model="checkPatientRestriction"/>
```

Comment:
Now that 'checkPatientRestriction' has been removed from the v-model, could we remove all references to it, in data(), resetForm(), and in setting the request object property. Could it just be defaulted either in the UI request object or on the API side?

Reply:
Agreed. I'll remove those references as well. I'll keep the response related code since it's harmless.